### ISSUE TITLE
Possible to have overflow:scroll for grid area?

### ISSUE THREAD

**Issue by dev-on2air** [nomcopter/react-mosaic#169]
Is it possible to have the mosaic area be larger than the display area so that you can have things taller than the window height and be able to scroll down or right?  I've tried a little playing with the css but not successful.

**Comment by nickfortyau** [comment]
Also needing this behavior as well

**Comment by Lonli-Lokli** [comment]
Yes, Mosaic just fills its parent, so give it a bigger parent inside a scroll container: https://nomcopter.github.io/react-mosaic/docs/guides/recipes#a-layout-bigger-than-the-screen
